lib/mapper.js

```diff
@@ -108,6 +113,53 @@ function formatCompany(company){
   return ret;
 }
 
+ /**
+ * Formats location data for use with intercom
```

Comment:
Accepts a context object and formats location/timezone information into an Intercom-compatible format